package/gluon-check-connection/luasrc/usr/bin/gluon-check-connection

```diff
@@ -7,14 +7,8 @@ local site = require 'gluon.site'
 local offline_flag_file = "/tmp/gluon_offline"
```

Comment:
Let's clean up names and paths a bit:
- Let's rename the script to `gluon-connection-check` (if we're going with my suggestion to rename the package)
- A better place for the flag file would be `/var/gluon/connection-check/offline`
- The cron job and micrond dependency can be moved from the domain switch package as well

Reply:
since we keep the name check-connectin now, I will move the flag to `/var/gluon/check-connection/offline`

I hope, that there will be no write-action done to the flash-rom there either. (Which was the reason, we chose /tmp to store the flag)

Reply:
I am not sure, how to move the cron from the scheduled domain switch to the new package, as the cron job should only run, in case domain switch is scheduled

Reply:
/tmp and /var are the same directory on OpenWrt, just symlinked.

Having two cronjobs, one in domain-switch and one in ssid-changer, isn't nice either. There would need to be some central facility in the check-connection package that other package can interface with to enable the cron job. Possibly a directory under /var/gluon where packages can drop a file when they need the offline status?